apps/web/src/index.css

```
@@ -0,0 +1,3 @@
+@tailwind base;
```

Biome (2.4.4)

[error] Unexpected unknown at-rule: tailwind 

(lint/suspicious/noUnknownAtRules)